File: src/main/java/by/owm/service/user/UserService.java
package by.owm.service.user;

import by.owm.model.User;

import java.util.List;

/**
 * Created by HaRiA on 30.07.2014.
 */
public interface UserService
{
    public void saveUser(User user);

    public List<User> userList();

    public void removeUser(long userId);

    public User returnUserByLoginPassword(String login, String password);

}

File: src/main/webapp/services/UserService.js
/**
 * Created by haria on 17.09.2015.
 */
angular.module('user.service', [])
    .service('userService', ['$http' , function ($http)
    {
        this.login = function(login,password) {
            var creditials = {};
            creditials.login = login;
            creditials.password = password;

            return $http.post('/rest/user/autenticate', JSON.stringify(creditials));
        };

        this.create_user = function(login, password)
        {
            var creditials = {};
            creditials.login = login;
            creditials.password = password;

            return $http.post('/rest/user/create', JSON.stringify(creditials));
        };


    }]);